diff --git a/src/gnome-cmd-user-actions.cc~ b/src/gnome-cmd-user-actions.cc
index caca3ea..3cce5b0 100644
--- a/src/gnome-cmd-user-actions.cc~
+++ b/src/gnome-cmd-user-actions.cc
@@ -1130,7 +1130,7 @@ void command_root_mode (GtkMenuItem *menuitem, gpointer not_used)
     argv[i++] = su;
     if (need_c)
         argv[i++] = "-c";
-    argv[i++] = g_get_prgname ();
+    argv[i++] = const_cast<char*>(g_get_prgname ());
     argv[i++] = NULL;
 
     GError *error = NULL;
