--- gnome-commander-1.2.8.15/libgcmd/libgcmd-widget-factory.cc~	2011-12-06 20:10:27.000000000 +0100
+++ gnome-commander-1.2.8.15/libgcmd/libgcmd-widget-factory.cc	2012-11-20 14:28:29.542265649 +0100
@@ -652,7 +652,7 @@
     va_end (argptr);
 
     dialog = gtk_message_dialog_new (GTK_WINDOW (main_win_widget), GTK_DIALOG_MODAL,
-                                     GTK_MESSAGE_ERROR, GTK_BUTTONS_OK, string);
+                                     GTK_MESSAGE_ERROR, GTK_BUTTONS_OK, "%s", string);
 
     gtk_signal_connect (GTK_OBJECT (dialog), "response", GTK_SIGNAL_FUNC (on_response), dialog);
 
@@ -673,7 +673,7 @@
 
     dialog = gtk_message_dialog_new (
         GTK_WINDOW (main_win_widget), GTK_DIALOG_MODAL,
-        GTK_MESSAGE_WARNING, GTK_BUTTONS_OK, msg);
+        GTK_MESSAGE_WARNING, GTK_BUTTONS_OK, "%s", msg);
 
     gtk_signal_connect (GTK_OBJECT (dialog), "response",
                         GTK_SIGNAL_FUNC (on_response), dialog);
--- gnome-commander-1.2.8.15/src/utils.h~	2011-12-06 20:10:29.000000000 +0100
+++ gnome-commander-1.2.8.15/src/utils.h	2012-11-20 14:29:31.845596785 +0100
@@ -256,10 +256,10 @@
                                              GTK_DIALOG_DESTROY_WITH_PARENT,
                                              GTK_MESSAGE_ERROR,
                                              GTK_BUTTONS_OK,
-                                             message.c_str());
+                                             "%s", message.c_str());
 
     if (secondary_text)
-        gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dlg), secondary_text);
+        gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dlg), "%s", secondary_text);
 
     gtk_dialog_run (GTK_DIALOG (dlg));
     gtk_widget_destroy (dlg);
--- gnome-commander-1.2.8.15/src/gnome-cmd-prepare-xfer-dialog.cc~	2011-12-06 20:10:28.000000000 +0100
+++ gnome-commander-1.2.8.15/src/gnome-cmd-prepare-xfer-dialog.cc	2012-11-20 14:31:12.308926576 +0100
@@ -192,7 +192,7 @@
                                                             (GtkDialogFlags) 0,
                                                             GTK_MESSAGE_QUESTION,
                                                             GTK_BUTTONS_OK_CANCEL,
-                                                            msg);
+                                                            "%s", msg);
                 gint choice = gtk_dialog_run (GTK_DIALOG (dialog));
                 gtk_widget_destroy (dialog);
                 g_free (msg);
--- gnome-commander-1.2.8.15/src/gnome-cmd-xml-config.cc~	2011-12-06 20:10:28.000000000 +0100
+++ gnome-commander-1.2.8.15/src/gnome-cmd-xml-config.cc	2012-11-20 14:31:58.785591601 +0100
@@ -546,7 +546,7 @@
     if (!g_markup_parse_context_parse (context, xml, xml_len, &error) ||
         !g_markup_parse_context_end_parse (context, &error))
     {
-        g_warning (error->message);
+        g_warning ("%s", error->message);
         g_error_free (error);
     }
 
@@ -567,7 +567,7 @@
 
     if (!g_file_get_contents (path, &xml, &xml_len, &error))
     {
-        g_warning (error->message);
+        g_warning ("%s", error->message);
         g_error_free (error);
 
         return FALSE;
--- gnome-commander-1.2.8.15/src/utils.cc~	2011-12-06 20:10:29.000000000 +0100
+++ gnome-commander-1.2.8.15/src/utils.cc	2012-11-20 14:32:47.258923226 +0100
@@ -193,7 +193,7 @@
     button_titles = convert_varargs_to_name_array (button_title_args);
     va_end (button_title_args);
 
-    dialog = gtk_message_dialog_new (GTK_WINDOW (main_win), GTK_DIALOG_MODAL, msg_type, GTK_BUTTONS_NONE, text);
+    dialog = gtk_message_dialog_new (GTK_WINDOW (main_win), GTK_DIALOG_MODAL, msg_type, GTK_BUTTONS_NONE, "%s", text);
     if (title)
         gtk_window_set_title (GTK_WINDOW (dialog), title);
 
@@ -590,7 +590,7 @@
         {
             gchar *msg = g_strdup_printf (_("%s does not know how to open remote file. Do you want to download the file to a temporary location and then open it?"), gnome_cmd_app_get_name (app));
             GtkWidget *dialog = gtk_message_dialog_new (GTK_WINDOW (main_win), GTK_DIALOG_MODAL,
-                                                        GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO, msg);
+                                                        GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO, "%s", msg);
             TmpDlData *dldata = g_new0 (TmpDlData, 1);
             args[0] = (gpointer) app;
             // args[2] is NULL here (don't set exec dir for temporarily downloaded files)
