--- gnome-commander-1.2.8.6/libgcmd/Makefile.am~	2010-03-04 21:06:22.000000000 +0100
+++ gnome-commander-1.2.8.6/libgcmd/Makefile.am	2010-06-13 19:06:12.175915557 +0200
@@ -3,7 +3,9 @@
 AM_CPPFLAGS = \
 	$(CC_WARNINGS) \
 	$(GNOMEUI_CFLAGS) \
-	$(GNOMEVFS_CFLAGS)
+	$(GNOMEVFS_CFLAGS) \
+	$(GNOME_CFLAGS) \
+	$(GTK_CFLAGS)
 
 pkglib_LTLIBRARIES = libgcmd.la
 
--- gnome-commander-1.2.8.6/src/Makefile.am~	2010-06-01 20:53:06.000000000 +0200
+++ gnome-commander-1.2.8.6/src/Makefile.am	2010-06-13 19:16:19.139206776 +0200
@@ -8,6 +8,7 @@
 	$(GNOMEUI_CFLAGS) \
 	$(GNOMEVFS_CFLAGS) \
 	$(PYTHON_CFLAGS) \
+	$(IPTCDATA_CFLAGS) \
 	-DDATADIR=\""$(datadir)"\"\
 	-DPLUGIN_DIR=\""$(libdir)/$(PACKAGE)/plugins"\"
 
@@ -111,6 +112,7 @@
 	$(GSF_LIBS) \
 	$(POPPLER_LIBS) \
 	$(PYTHON_LIBS) \
-	$(PYTHON_EXTRA_LIBS)
+	$(PYTHON_EXTRA_LIBS) \
+	$(IPTCDATA_LIBS)
 
 gcmd_block_SOURCES = block.cc
